manifests/init.pp

```diff
@@ -183,112 +178,100 @@
+  Optional[Variant[Numeric, String]] $package_apt_pin              = undef,
+  String $package_ensure                                           = $rabbitmq::params::package_ensure,
+  String $package_gpg_key                                          = $rabbitmq::params::package_gpg_key,
+  String $package_name                                             = $rabbitmq::params::package_name,
+  Optional[String] $package_source                                 = undef,
+  Optional[String] $package_provider                               = undef,
+  Boolean $repos_ensure                                            = $rabbitmq::params::repos_ensure,
+  Boolean $manage_python                                           = $rabbitmq::params::manage_python,
+  String $rabbitmq_user                                            = $rabbitmq::params::rabbitmq_user,
+  String $rabbitmq_group                                           = $rabbitmq::params::rabbitmq_group,
+  String $rabbitmq_home                                            = $rabbitmq::params::rabbitmq_home,
+  Integer $port                                                    = $rabbitmq::params::port,
+  Boolean $tcp_keepalive                                           = $rabbitmq::params::tcp_keepalive,
+  Integer $tcp_backlog                                             = $rabbitmq::params::tcp_backlog,
+  Optional[Integer] $tcp_sndbuf                                    = undef,
+  Optional[Integer] $tcp_recbuf                                    = undef,
+  Optional[Integer] $heartbeat                                     = undef,
+  Enum['running', 'stopped'] $service_ensure                       = $rabbitmq::params::service_ensure,
+  Boolean $service_manage                                          = $rabbitmq::params::service_manage,
+  String $service_name                                             = $rabbitmq::params::service_name,
+  Boolean $ssl                                                     = $rabbitmq::params::ssl,
+  Boolean $ssl_only                                                = $rabbitmq::params::ssl_only,
+  Optional[String] $ssl_cacert                                     = undef,
+  Optional[String] $ssl_cert                                       = undef,
+  Optional[String] $ssl_key                                        = undef,
+  Optional[Integer] $ssl_depth                                     = undef,
+  Optional[String] $ssl_cert_password                              = undef,
+  Integer $ssl_port                                                = $rabbitmq::params::ssl_port,
+  Optional[String] $ssl_interface                                  = undef,
+  Integer $ssl_management_port                                     = $rabbitmq::params::ssl_management_port,
+  Integer $ssl_stomp_port                                          = $rabbitmq::params::ssl_stomp_port,
+  String $ssl_verify                                               = $rabbitmq::params::ssl_verify,
```

Comment:
I *think* this could be `Enum['verify_none','verify_peer']`??  See http://erlang.org/doc/man/ssl.html